backend/apps/owasp/management/commands/slack_sync_conversation_list.py

```diff
@@ -0,0 +1,274 @@
+                    "creator_id",
+                )
+            }
+
+        # Track conversations to create or update
+        to_create = []
+        to_update = []
+
+        # Pagination variables
+        next_cursor = None
+        total_processed = 0
+
+        while True:
+            try:
+                # Fetch batch of conversations
+                result = app.client.conversations_list(
+                    limit=batch_size,
+                    exclude_archived=False,
+                    types="public_channel,private_channel",
+                    cursor=next_cursor or None,
+                )
+
+                if not result.get("ok"):
+                    error = result.get("error", "Unknown error")
+                    handle_api_error(error)
+
+                conversations = result.get("channels", [])
+
+                # Process this batch
+                if not dry_run:
+                    new_creates, new_updates = self._prepare_conversation_changes(
+                        conversations, conversation_model, existing_channels
+                    )
+                    to_create.extend(new_creates)
+                    to_update.extend(new_updates)
+
+                total_processed += len(conversations)
+                self.stdout.write(f"Processed {len(conversations)} conversations...")
+
+                # Handle pagination
+                next_cursor = result.get("response_metadata", {}).get("next_cursor")
+                if not next_cursor:
+                    break
+
+                # Rate limiting protection
+                if delay > 0:
+                    time.sleep(delay)
+
+            except SlackApiError as e:
+                handle_api_error(e, f"Error calling Slack API: {e}")
+            except (ValueError, TypeError, ConnectionError) as e:
+                handle_api_error(e, f"Unexpected error: {e}")
+
+        # Perform bulk operations
+        if not dry_run and (to_create or to_update):
+            with transaction.atomic():
+                if to_create:
+                    conversation_model.objects.bulk_create(to_create)
+                    logger.info("Bulk created %d conversations", len(to_create))  # G004 fix
+
+                if to_update:
+                    conversation_model.objects.bulk_update(
+                        to_update,
+                        [
+                            "name",
+                            "created_at",
+                            "is_private",
+                            "is_archived",
+                            "is_general",
+                            "topic",
+                            "purpose",
+                            "creator_id",
+                        ],
+                    )
+                    logger.info("Bulk updated %d conversations", len(to_update))  # G004 fix
+
+        return total_processed
+
+    def _prepare_conversation_changes(self, conversations, conversation_model, existing_channels):
```

Comment:
This has to be a part of the Conversation model. Also implement based on existing examples.